pkg/Microsoft.Windows.Compatibility/Microsoft.Windows.Compatibility.pkgproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup>
     <PrereleaseLibraryPackage>
-      <Version>4.5.0</Version>
+      <Version>4.6.0</Version>
```

Comment:
Can you have this start to us the property in packaging.props? https://github.com/dotnet/corefx/blob/master/Packaging.props#L36 (somehow :))

Reply:
I ran a /pp on the compatibility pkgproj and it is already imported by pkg/dir.props -- will update to just use the property :)

Reply:
Note that it gets overridden in this project so you will need to capture the value of PackageVersion before setting the value to 2.1.0

Reply:
Fixed.